.github/workflows/pytest-cov.yaml

```diff
@@ -1,49 +1,56 @@
 name: Pytest Coverage
-on: 
+on:
   push:
-    branches: [ develop ]
+    branches: [develop, 53-create-git-actions-workflow-pytest-cov]
   pull_request:
-    branches: [ develop ]
+    branches: [develop]
 
 jobs:
   run_pytest_coverage:
-    runs-on: ubuntu-latest
     name: Run Pytest Coverage
-    strategy:
-      matrix:
-        python: ["3.7"]
-
+    runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Setup Python
-        uses: actions/setup-python@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v1
         with:
-          python-version: ${{ matrix.python }}
+          python-version: "3.10"
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
 
-      - name: Install dependencies
+      - name: Install App
         run: |
-          pip install --upgrade pip
+          python -m pip install --upgrade pip
           pip install -e .[dev]
 
       - name: Run pytest-cov
         run: |
-          pytest --cov=src/uwtools/ --cov-fail-under=5 --cov-report=html:cov_html
+          pytest --cov-fail-under=5 --cov-report=html --cov=uwtools
```

Comment:
Is this 5% as a min? The docs are not very clear on what that unit is. 

If 5%, do we want to shoot for something a little higher since we're seeing pretty good coverage already? Maybe we could just set it to like 80% so we get informed if it drops too much lower than the current levels?

Reply:
I was just suggesting a placeholder that would be a tad more useful now. If we choose something that is just below our current level, we won't be dropping too low in the interim before we know what the true desired value will be.

Reply:
Changed to 70%